packages/lesswrong/components/common/KarmaDisplay.tsx

```diff
@@ -14,13 +20,26 @@ const KarmaDisplay = ({document, placement="left"}: {
     ? document.afBaseScore
     : null;
   const {LWTooltip} = Components;
+
+  const {document: comment} = useSingle({
+    documentId: annualReviewMarketCommentId,
+    collectionName: "Comments",
+    fragmentName: "CommentsListWithParentMetadata",
+    skip: !annualReviewMarketCommentId,
+  })
```

Comment:
This is unfortunately the sort of thing that can have pretty negative performance impacts; `KarmaInfo` is used in e.g. `LWPostsItem` which we render many instances of many pages (i.e. the home page).  This sends off an individual query per post item in the middle of the rendering pass.

(I think for now we just want to put a TODO here to see if we can refactor it after https://github.com/ForumMagnum/ForumMagnum/pull/8314 gets merged in.)